2023-01-19 05:22:20,735 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 11.8, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Test1', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {})}, client_id=Test1@33022dd4997fc1a641dfd9c808c89aaa6fceb710, version_check=False)
2023-01-19 05:22:20,737 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:22:20,738 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:22:20.737009", "client_id": "b1cec76a18f18556508bcbe25dc4fe90e3d321bad94981841a5299c0eb2cc7bdb8fa1769129ae4be6795b72f90a647d70b461596a6781041ec2061b44183f155", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 11.8, "potential_savings": 0.5, "report_savings": true, "suite_name": "Test1", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:22:20,891 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:22:20,893 Caching directory does not exist! Creating it...
2023-01-19 05:22:21,565 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:22:21,566 aide.submit_statistics() returned: False
2023-01-19 05:34:31,568 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.1, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Test2', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {})}, client_id=Test2@eacc10941135d39105593d5d35a385878b10be1e, version_check=False)
2023-01-19 05:34:31,569 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:34:31,569 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:34:31.569259", "client_id": "837962c8cb73131b03d0f9fc60be0e8641d05d218028203347919ad60aec644c1d70444f48ff8309955d2a299c532fc5c78d05a2bc4807efe55732759268259d", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.1, "potential_savings": 0.5, "report_savings": true, "suite_name": "Test2", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:34:31,673 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:34:31,675 Telemetry cache has contents!
2023-01-19 05:34:32,362 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:34:32,363 aide.submit_statistics() returned: False
2023-01-19 05:41:18,428 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.6, 'potential_savings': 1.1, 'report_savings': True, 'suite_name': 'Loops', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {})}, client_id=Loops@c3b9980a2e129a2c62ea29cf458d12e3a8624ebb, version_check=False)
2023-01-19 05:41:18,430 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:41:18,431 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:41:18.429938", "client_id": "499a69dc31a6666e2f59d1a95192a63a640e920c128d95775e6625b6f323d9acb4ec5f1a82209c48a22db4ece0b16c381231d98a5a7b3ea2ab4cb93c3c6d9a8b", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.6, "potential_savings": 1.1, "report_savings": true, "suite_name": "Loops", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:41:18,537 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:41:18,539 Telemetry cache has contents!
2023-01-19 05:41:19,160 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:41:19,162 aide.submit_statistics() returned: False
2023-01-19 05:47:55,762 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.2, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Keywords', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {})}, client_id=Keywords@0f3ae25eaabd9e4bae2e9d7a905d3abecbc759f3, version_check=False)
2023-01-19 05:47:55,763 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:47:55,764 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:47:55.763599", "client_id": "4a7ae37fae281c256d69f8c996da7b59d52cd0fb290deaae989ef8b75c34c7fd7a09fc3aa00bb0ec479580dcc172d03bcecdd3074bc5a922a23b1c725c646709", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.2, "potential_savings": 0.5, "report_savings": true, "suite_name": "Keywords", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:47:55,870 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:47:55,872 Telemetry cache has contents!
2023-01-19 05:47:56,545 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:47:56,546 aide.submit_statistics() returned: False
2023-01-19 05:52:53,361 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.0, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Keywords', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {})}, client_id=Keywords@0f3ae25eaabd9e4bae2e9d7a905d3abecbc759f3, version_check=False)
2023-01-19 05:52:53,362 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:52:53,363 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:52:53.362254", "client_id": "4a7ae37fae281c256d69f8c996da7b59d52cd0fb290deaae989ef8b75c34c7fd7a09fc3aa00bb0ec479580dcc172d03bcecdd3074bc5a922a23b1c725c646709", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.0, "potential_savings": 0.5, "report_savings": true, "suite_name": "Keywords", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:52:53,518 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:52:53,520 Telemetry cache has contents!
2023-01-19 05:52:54,168 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:52:54,169 aide.submit_statistics() returned: False
2023-01-19 05:53:05,956 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 1.8, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 1})}, client_id=Device Connectivity@97d78a5abca5df1e627299e102037768bac41082, version_check=False)
2023-01-19 05:53:05,957 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 05:53:05,958 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 05:53:05.957197", "client_id": "9a0981b321bdce1dd097e4fc3162023439ee5433edcf4a2f6aa6ffe49717abfea5badcb7ad39ef7dff04ee1e7417458a119b7f1ab5443115e2b62866ed4a5529", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 1.8, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 05:53:06,064 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 05:53:06,065 Telemetry cache has contents!
2023-01-19 05:53:06,661 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 05:53:06,662 aide.submit_statistics() returned: False
2023-01-19 06:01:31,179 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.1, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 1})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:01:31,182 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:01:31,183 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:01:31.181789", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.1, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:01:31,295 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:01:31,297 Telemetry cache has contents!
2023-01-19 06:01:31,959 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:01:31,961 aide.submit_statistics() returned: False
2023-01-19 06:02:46,430 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 3.1, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 8, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:02:46,431 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:02:46,432 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:02:46.431417", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 3.1, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 8, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:02:46,540 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:02:46,542 Telemetry cache has contents!
2023-01-19 06:02:47,212 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:02:47,214 aide.submit_statistics() returned: False
2023-01-19 06:04:05,251 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 13.2, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 11, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:04:05,253 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:04:05,254 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:04:05.252984", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 13.2, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 11, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:04:05,359 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:04:05,361 Telemetry cache has contents!
2023-01-19 06:04:06,036 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:04:06,037 aide.submit_statistics() returned: False
2023-01-19 06:06:55,781 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.2, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 10, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:06:55,783 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:06:55,784 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:06:55.783149", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.2, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 10, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:06:55,889 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:06:55,891 Telemetry cache has contents!
2023-01-19 06:06:56,551 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:06:56,553 aide.submit_statistics() returned: False
2023-01-19 06:11:44,336 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 14.0, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 13, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:11:44,338 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:11:44,339 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:11:44.337925", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 14.0, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 13, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:11:44,446 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:11:44,449 Telemetry cache has contents!
2023-01-19 06:11:45,104 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:11:45,106 aide.submit_statistics() returned: False
2023-01-19 06:12:57,628 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 21.9, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 15, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:12:57,629 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:12:57,631 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:12:57.629803", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 21.9, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 15, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:12:57,737 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:12:57,739 Telemetry cache has contents!
2023-01-19 06:12:58,403 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:12:58,405 aide.submit_statistics() returned: False
2023-01-19 06:14:16,124 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 34.3, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 3})}, client_id=Device Connectivity@bb99a2639f460fed663545545b2b10a997439e9a, version_check=False)
2023-01-19 06:14:16,126 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:14:16,127 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:14:16.126183", "client_id": "5b12e900b923b5e8937f2b6eba89fc7a70b2c763839962dab0118e4817c3d0d6787953e5db10dec89aab12781bd368be333111e0f7a63dcb3ba0386dd4fde52d", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 34.3, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 3}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:14:16,234 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:14:16,236 Telemetry cache has contents!
2023-01-19 06:14:16,894 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:14:16,895 aide.submit_statistics() returned: False
2023-01-19 06:16:30,545 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 14.1, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 13, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:16:30,547 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:16:30,548 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:16:30.547272", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 14.1, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 13, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:16:30,665 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:16:30,667 Telemetry cache has contents!
2023-01-19 06:16:31,354 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:16:31,355 aide.submit_statistics() returned: False
2023-01-19 06:17:00,489 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.3, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'DeviceCli': 1})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:17:00,490 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:17:00,491 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:17:00.490348", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.3, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:17:00,605 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:17:00,609 Telemetry cache has contents!
2023-01-19 06:17:01,349 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:17:01,351 aide.submit_statistics() returned: False
2023-01-19 06:18:02,245 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 35.5, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 3})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:18:02,246 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:18:02,248 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:18:02.246693", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 35.5, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 3}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:18:02,366 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:18:02,368 Telemetry cache has contents!
2023-01-19 06:18:02,979 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:18:02,981 aide.submit_statistics() returned: False
2023-01-19 06:19:14,368 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 22.4, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Device Connectivity', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 13, 'Parsing': 2})}, client_id=Device Connectivity@d260cc8572d1a0031da3ac6916c95028298cc214, version_check=False)
2023-01-19 06:19:14,369 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:19:14,370 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:19:14.369479", "client_id": "7a863833b23bc1e6a45f7d9ae5fbcca31f151b90c5549528ca6789ba88dcf86d2c070e44abe54d325447c6ceba5ea75d2defb9419458469992bea44128ca0e34", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 22.4, "potential_savings": 0.5, "report_savings": true, "suite_name": "Device Connectivity", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 13, "Parsing": 2}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:19:14,490 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:19:14,494 Telemetry cache has contents!
2023-01-19 06:19:15,189 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:19:15,191 aide.submit_statistics() returned: False
2023-01-19 06:21:57,862 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 1.8, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Conditional', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 1})}, client_id=Conditional@cdb31a47e6bf3462cdcfe0f7c16713bd5d89faed, version_check=False)
2023-01-19 06:21:57,864 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:21:57,865 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:21:57.864474", "client_id": "44a2fe706b6dcfe7bb26d6db7dcaebac0475d6febedce400c059e6fb668cf548bfc4395bf64a74b4c1732ac0ac7c7e6e912157bfe6b413fcb47c5d4665bfa9c6", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 1.8, "potential_savings": 0.5, "report_savings": true, "suite_name": "Conditional", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:21:57,978 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:21:57,980 Telemetry cache has contents!
2023-01-19 06:21:58,660 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:21:58,662 aide.submit_statistics() returned: False
2023-01-19 06:23:23,813 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 2.0, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Conditional', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 1})}, client_id=Conditional@cdb31a47e6bf3462cdcfe0f7c16713bd5d89faed, version_check=False)
2023-01-19 06:23:23,814 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:23:23,816 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:23:23.814395", "client_id": "44a2fe706b6dcfe7bb26d6db7dcaebac0475d6febedce400c059e6fb668cf548bfc4395bf64a74b4c1732ac0ac7c7e6e912157bfe6b413fcb47c5d4665bfa9c6", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 2.0, "potential_savings": 0.5, "report_savings": true, "suite_name": "Conditional", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:23:23,985 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:23:23,987 Telemetry cache has contents!
2023-01-19 06:23:24,619 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:23:24,621 aide.submit_statistics() returned: False
2023-01-19 06:24:02,384 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 1.8, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Conditional', 'run_from': 'cli', 'devices_os': {}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 1})}, client_id=Conditional@cdb31a47e6bf3462cdcfe0f7c16713bd5d89faed, version_check=False)
2023-01-19 06:24:02,385 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:24:02,387 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:24:02.385543", "client_id": "44a2fe706b6dcfe7bb26d6db7dcaebac0475d6febedce400c059e6fb668cf548bfc4395bf64a74b4c1732ac0ac7c7e6e912157bfe6b413fcb47c5d4665bfa9c6", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 1.8, "potential_savings": 0.5, "report_savings": true, "suite_name": "Conditional", "run_from": "cli", "devices_os": {}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 1}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:24:02,501 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:24:02,503 Telemetry cache has contents!
2023-01-19 06:24:03,154 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:24:03,155 aide.submit_statistics() returned: False
2023-01-19 06:28:42,187 CallHome: calling aide.submit_statistics(url=None, tool_id=52380, metadata={'pid': '000000', 'customer': '', 'cxta_version': '22.24', 'execution_time': 11.5, 'potential_savings': 0.5, 'report_savings': True, 'suite_name': 'Conditional', 'run_from': 'cli', 'devices_os': {'iosxr': 4, 'iosxe': 1, 'None': 2}, 'cxtalibs_used': defaultdict(<class 'int'>, {'Testbed': 1, 'DeviceCli': 6, 'Parsing': 4})}, client_id=Conditional@20e0d4319f36054d287f7e372c1e49b51467c83b, version_check=False)
2023-01-19 06:28:42,189 The CX Project ID will be collected for AIDE reporting purposes only.
2023-01-19 06:28:42,190 JSON data to be sent: '[{"aide_uid": "52380", "timestamp": "2023-01-19 06:28:42.188870", "client_id": "69f2daa6c23265d52c4650e1476500ba6febd6423b97e09e11be039445fc6b036896c7bacef60cf54fe302c121d957826dad40045237f24a9f5b98383a40eb7e", "cli": "cxta", "language": "python", "language_version": "3.9.15", "pid": "000000", "metadata": {"pid": "000000", "customer": "", "cxta_version": "22.24", "execution_time": 11.5, "potential_savings": 0.5, "report_savings": true, "suite_name": "Conditional", "run_from": "cli", "devices_os": {"iosxr": 4, "iosxe": 1, "None": 2}, "cxtalibs_used": {"Testbed": 1, "DeviceCli": 6, "Parsing": 4}}, "os": "Linux", "agent_name": "aide-python-agent", "agent_version": "4.1.0", "os_version": "5.10.102.1-microsoft-standard-WSL2"}]'
2023-01-19 06:28:42,352 Using default server 'https://api-auc.cisco.com:443/api/collect/'
2023-01-19 06:28:42,356 Telemetry cache has contents!
2023-01-19 06:28:42,950 The following error occurred: 'HTTPSConnectionPool(host='api-auc.cisco.com', port=443): Read timed out. (read timeout=0.5)'
2023-01-19 06:28:42,952 aide.submit_statistics() returned: False
